src/modules/dashboard/components/Extensions/ExtensionDetails.tsx

```diff
@@ -375,7 +375,10 @@ const ExtensionDetails = ({
                   )}
                   {extensionEnabled &&
                     extension.enabledExtensionBody &&
-                    extension.enabledExtensionBody()}
+                    extension.initializationParams &&
+                    extension.enabledExtensionBody(
+                      extension.initializationParams,
+                    )}
```

Comment:
Why are the initialization params passed here to the extension body ?

Params should be visible only when enabling the extension, the body is visible **after** you enable it

Reply:
Yeah, this doesn't work. At all!

You need the initialization params available on the extension enable page, otherwise the values won't be available to you to be able to send to the "enableExtension" saga.